.github/workflows/build_and_test.yaml

```diff
@@ -16,31 +16,51 @@ on:
       - "**/*.png"
 env:
   GO_VERSION: 1.18.2
+  ENVOY_GATEWAY_DEV_IMAGE: envoyproxy/gateway-dev
+  ENVOY_GATEWAY_DEV_TAG: latest
 jobs:
-  build:
+  build-and-test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
           go-version: ${{ env.GO_VERSION }}
           cache: true
-      - name: build
-        run: make build
-  test:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
-          cache: true
-      - name: test and report coverage
+      - name: Test and report coverage
         run: go test ./... -race -coverprofile=coverage.xml -covermode=atomic
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
         with:
           fail_ci_if_error: true
           files: ./coverage.xml
           name: codecov-envoy-gateway
-          verbose: true
\ No newline at end of file
+          verbose: true
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
I think we can drop this since we're only building a single arch now and when we add support for multi-arch images we can use Go's native cross-compilation

Reply:
based on the comments, added `linux/arm64` to enable `envoy-gateway` images to run on `arm64` as well (e.g. M1 Macbooks)